modules/Kconfig

```diff
@@ -30,3 +34,23 @@ source "modules/Kconfig.tinycbor"
 source "modules/Kconfig.tinycrypt"
 source "modules/Kconfig.vega"
 source "modules/Kconfig.xtensa"
+
+comment "Unavailable modules, please install those via the project manifest."
+
+# List of Zephyr modules text when module is not available, please use the
+# following syntax:
+# ---------------------------------------------------
+# comment "<module_name> module not available."
```

Comment:
Could a template be used for this?

Reply:
Then you would end with three lines instead of two, like:
```
module_name=<module_name>
MODULE_NAME_UPPER=<MODULE_NAME_UPPER>
source 'comment.template'
```
plus the template code file, instead of two lines.

I don't see the benefit in that.